SwiftVideoBackground/ViewController.swift

```diff
@@ -0,0 +1,11 @@
+import UIKit
+
+class ViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
```

Comment:
Can we add this here?:

```swift
override var prefersStatusBarHidden: Bool { return true }
```

Reply:
Does this hide the status bar? If so I don't think this is needed as it currently is not showing.

Reply:
Ah okay never mind my bad haha.

Reply:
✅